### PR TITLE
Chrylser: move to platform config

### PR DESCRIPTION
**Pull request by jnewb1**

### CODE REVIEW

File: selfdrive/car/chrysler/interface.py [hunk 1]
```diff
@@ -24,7 +24,6 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
     elif candidate in RAM_DT:
       ret.safetyConfigs[0].safetyParam |= Panda.FLAG_CHRYSLER_RAM_DT
 
-    ret.minSteerSpeed = 3.8  # m/s
```
Comment (sshane) [inline selfdrive/car/chrysler/interface.py:27]: when is this set?

Comment (jnewb1) [reply selfdrive/car/chrysler/interface.py:27]: https://github.com/commaai/openpilot/blob/master/selfdrive/car/interfaces.py#L123

File: selfdrive/car/chrysler/values.py [hunk 2]
```diff
@@ -13,25 +13,89 @@
...
+
+
+@dataclass
+class ChryslerPlatformConfig(PlatformConfig):
+  dbc_dict: DbcDict = field(default_factory=lambda: dbc_dict('chrysler_pacifica_2017_hybrid_generated', None))
+
+
+@dataclass
+class ChryslerCarSpecs(CarSpecs):
+  minSteerSpeed: float = 3.8 # m/s
```
Comment (sshane) [inline selfdrive/car/chrysler/values.py:29]:
```suggestion
  minSteerSpeed: float = 3.8  # m/s
```

File: selfdrive/car/chrysler/values.py [hunk 3]
```diff
@@ -13,25 +13,89 @@
...
+  DODGE_DURANGO = ChryslerPlatformConfig(
+    "DODGE DURANGO 2021",
+    ChryslerCarInfo("Dodge Durango 2020-21"),
+    specs=PACIFICA_2017_HYBRID.specs,
+  )
 
   # Jeep
-  JEEP_GRAND_CHEROKEE = "JEEP GRAND CHEROKEE V6 2018"  # includes 2017 Trailhawk
-  JEEP_GRAND_CHEROKEE_2019 = "JEEP GRAND CHEROKEE 2019"  # includes 2020 Trailhawk
+  JEEP_GRAND_CHEROKEE = ChryslerPlatformConfig( # includes 2017 Trailhawk
```
Comment (sshane) [inline selfdrive/car/chrysler/values.py:71]:
```suggestion
  JEEP_GRAND_CHEROKEE = ChryslerPlatformConfig(  # includes 2017 Trailhawk
```

**Pull request merged**